cfme/fixtures/v2v_fixture.py

```diff
@@ -0,0 +1,503 @@
+import fauxfactory
+import itertools
+import pytest
+import json
+from collections import namedtuple
+from riggerlib import recursive_update
+from widgetastic.utils import partial_match
+
+from cfme.fixtures.provider import setup_or_skip
+from cfme.infrastructure.provider.rhevm import RHEVMProvider
+from cfme.infrastructure.provider.virtualcenter import VMwareProvider
+from cfme.utils.generators import random_vm_name
+from cfme.utils.log import logger
+
+
+FormDataVmObj = namedtuple("FormDataVmObj", ["form_data", "vm_list"])
+
+
+@pytest.fixture(scope="session")
```

Comment:
@sshveta we can remove `v2v_downstream_enable` check, since we are releasing IMS 1.0 in 5.10 GA and we don't have any plan to hide menu in future.